gluu-server/opt/gluu/jetty/oxauth/custom/pages/WEB-INF/incl/components/head-en.xhtml

```diff
@@ -10,12 +10,14 @@
     <meta name="dcterms.language" content="eng" />
     <meta name="dcterms.subject" content="#{title != null ? title : 'Sign In'}" />
     <meta name="dcterms.issued" content="2020-02-21" />
-    <meta name="dcterms.modified" content="2022-01-07" />
+    <meta name="dcterms.modified" content="2022-07-28" />
     <meta name="dcterms.spatial" content="Canada" />
     <meta name="dcterms.type" content="navigation page - topic page" />
     <meta name="dcterms.identifier" content="Sign_In_Canada" />
 
-    <title>#{title != null ? title : "Sign In"}</title>
+    <title>#{title != null ? title : "Sign In to"}
```

Comment:
Space after "Sign In to" ("Sign In to ") ? 
Same for "Se connecter à "

Reply:
There is a newline as whitespace.

Reply:
Ah ok, if that works then great !